11_rtc_driver/ds3231_regs.h

```diff
@@ -0,0 +1,29 @@
+#ifndef DS3231_REGS_H
+#define DS3231_REGS_H
+
+#define DS3231_REG_SECS		0x00
+#define DS3231_REG_MIN		0x01
+#define DS3231_REG_HOUR		0x02
+#	define DS3231_BIT_12HR		0x40
+#	define DS3231_BIT_PM		0x20
+#define DS3231_REG_WDAY		0x03
+#define DS3231_REG_MDAY		0x04
+#define DS3231_REG_MONTH	0x05
+#	define DS3231_BIT_CENTURY	0x80
+#define DS3231_REG_YEAR		0x06
+
```

Comment:
`#define DS3231_TIME_REGS 7` and use it instead of `7` in array declarations

Reply:
Done